README.md

```
@@ -1,58 +1,204 @@
+npm test
+
+# Run E2E tests
+npm run test:e2e
+
+# Build for production (verify no errors)
+npm run build
+```
+
+## 📋 Project Structure
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)